corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -933,13 +937,14 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
                 L.mapbox.accessToken = token;
 
                 const allCoordinates = [];
+                const nullCoordinate = ['0', '0'];
                 const markers = [];
                 this.options.collection.models
                     .forEach(model => {
                         const addressCoordinates = model.attributes.data[addressIndex];
                         if (addressCoordinates) {
                             let markerCoordinates = addressCoordinates.split(" ").slice(0,2);
-                            if (markerCoordinates.length > 1) {
+                            if (markerCoordinates.length > 1 && JSON.stringify(markerCoordinates) !== JSON.stringify(nullCoordinate)) {
```

Comment:
When does this case happen?

Reply:
That's a good question. If a geocoder question is left blank, then the property is just empty - so I don't know how this comes up. I assumed it was directly assigned vs via a geocoder question. 

There is a case for BHA which does have a "0 0" coordinate, which is why this specifically came up in the request ticket. I asked for clarification [here](https://dimagi.atlassian.net/browse/SUPPORT-22079?focusedCommentId=354264).

Reply:
Thanks for bringing this up. Delivery has agreed to clean up "0 0" values and make it blank. Also, future data will be blank instead of using "0 0" as "0 0" should be treated as a valid coordinate. Opened this [PR](https://github.com/dimagi/commcare-hq/pull/35369) to revert this specific change